presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -2,6 +2,30 @@
 Deploying Presto
 ================
 
+Requirements
```

Comment:
Not sure if we should call it Software Requirements

Reply:
Also maybe we should say that these requirements apply to Presto. And later add separate requirements for the JDBC driver and the CLI. I intend to expand both these pages/docs in the future..

Reply:
JDBC is in another PR. I will send another one for CLI.